### DISCUSSION TITLE
ModelCheckpoint and WandbLogger: set a common checkpoint filename

### DISCUSSION THREAD

**Discussion by sh3rlock14** [Lightning-AI/pytorch-lightning#16965] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hey everyone 👋🏻!

In `pytorch_lightning.callbacks.ModelCheckpoint` the `filename` parameter allows to set an highly customized filename for the checkpoint being logged, and this is super!🚀... but is it possible to pass this filename to `pytorch_lightning.loggers.WandbLogger` `checkpoint_name` parameter?

To have access to `wandb.config` I initialize WandbLogger first and ModelChekpoint then, and I would like to keep this configuration; so, since both the 'callbacks' and the 'logger' are passed to a `Trainer`, wouldn't it be possible for it to update the logger's checkpoint filename with ModelCheckpoint one?

Thanks in advance for your help! 🙏🏻

**Comment by royvelich** [comment]
This is really important... any workaround?
